lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -1,4 +1,4 @@
-local wire_expression2_entity_trails = CreateConVar("wire_expression2_entity_trails", 50, FCVAR_ARCHIVE, "Amount of trails one E2 can make. (0 - to disable trails)", 0)
+local wire_expression2_entity_trails_max = CreateConVar("wire_expression2_entity_trails_max", 30, FCVAR_ARCHIVE, "Max amount of trails all E2 can make. 0 - to disable trails. (Limit is shared between E2s)", 0)
```

Comment:
I think it should be a per player limit rather than completely global.

Reply:
It is xD. Will fix desc a bit later